esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3224,12 +3224,11 @@ bool CFileSprayEx::onDropZoneFiles(IEspContext &context, IEspDropZoneFilesReques
 
                 IpAddress ipAddr;
                 ipAddr.ipset(server.str());
```

Comment:
Why is the check for ipToMatch done via an ip address, instead of comparing the names?

Reply:
The names may not be same (like 'localhome', '.', or '127.0.0.1').

Reply:
But is that ever a problem, since for ecl watch the list has been obtained by calling a service that returns the entries from the environment.
The reason for raising it as an issue is I want to add dropzone targets that do not have associated ips.  The most natural way to support that would be to specify a pseudo-host name, but that will fail if all names are converted to ips.
@jakesmith @GordonSmith what are your thoughts?

Reply:
These ESP methods potentially get called from the client tools as well (not 100% if thie method is used specifically).

Suggestion:  Check if name matches via string compare - if not, fall back to IP4 compare.

Reply:
I don't know why IEspDropZoneFilesRequest uses a netaddres which is currently an IP..
What calls this, passing an IP now?

agree would be better if names, even with bare metal hostnames where he IP could be fluid.

Reply:
> What calls this, passing an IP now?
It could be called by any client through ECLWatch or SOAP call.

Reply:
> Suggestion: Check if name matches via string compare - if not, fall back to IP4 compare.

Does it work? For example, we have to 2 dropzone targets in the list. The first dropzone target has a pseudo-host name. The one for the ipToMatch is the second dropzone target. When comparing it with the first dropzone target, the strings are not the same. And we cannot do the IP4 comparison because the pseudo-host name cannot be converted to ip.

Reply:
Do we allow two pseudo-host names to point to the same dropzone target?

Reply:
Is it possible to use some flag inside the pseudo-host names to identify them as pseudo-host?

Reply:
It could check if the file path was a url, and avoid resolving it if so.  I am going to create a entry in the design discussion, because I think it is worth discussing more generally.